Feature/#508 change pipeline config documentation pictures (#547)
Avaiga/taipy-doc: docs/manuals/core/config/job-config.md

> With the _development_ mode, the jobs are synchronously executed one by one. The jobs are directly executed

Reviewer: ```suggestion
With the _development_ mode, the jobs are synchronously executed individually. The jobs are directly executed
```

in the main thread at the submission. Note that with the _development_ mode, the submit method waits for the

> jobs to be finished before to return. Please refer to the

Reviewer: ```suggestion
jobs to be finished before returning.
```

Reviewer: My suggestion is: 

For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).

Author: Haha yep I just saw it below! Sorry for the trouble! Have fixed and pushed the latest changes :D

Reviewer: Ok cool, thanks! :)

> [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section to
> the see how to submit jobs.

Reviewer: ```suggestion
For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).
```

executed.
Note that with the _standalone_ mode, the submit method in not blocking and returns after the job is queued.

> It means the submit method can return before the job finishes or even before it is dequeued. Please refer to

Reviewer: ```suggestion
It means the submit method can return before the job finishes or even before it is dequeued. 
```

> the [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section
> to the see how to submit jobs.

Reviewer: ```suggestion
For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).
```
